### PR TITLE
add an example for amp-sidebar #128

### LINKED ISSUE #128: Create an amp-sidebar example
Create an amp-sidebar example
https://github.com/ampproject/amphtml/blob/master/extensions/amp-sidebar/amp-sidebar.md

### PR DESCRIPTION
**Pull request by juliantoledo**
Resolves #128

### CODE REVIEW

File: src/20_Components/amp-sidebar.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<!---{
+    "experiment": true,
+    "component": "amp-sidebar"
+  }--->
+<!--
+  #### Introduction
+
+  A sidebar provides a way to display meta content intended for temporary access (navigation links, buttons, menus, etc.).The sidebar can be revealed by a button tap while the main content remains visually underneath.
```
Comment (sebastianbenz) [inline src/20_Components/amp-sidebar.html]: nit: s/.The/. The

File: src/20_Components/amp-sidebar.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    }
+  </style>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    There can be only one `amp-sidebar` in an AMP document. The `amp-sidebar` should be a direct child of the `<body>`.
+
+    The`amp-sidebar` may not contain any AMP Elements except for `amp-accordion`, `amp-img` and `amp-fit-text`.
```
Comment (sebastianbenz) [inline src/20_Components/amp-sidebar.html]: Can you add comment why amp-fit-text in a menu.

File: src/20_Components/amp-sidebar.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+          <amp-fit-text width="220" height="20" layout="responsive" max-font-size="24">
+            Nav item 3 - &lt;amp-fit-text&gt; longer text
+          </amp-fit-text>
+        </li>
+        <li> Nav item 4 - Image<amp-img class='amp-sidebar-image' src="/img/favicon.png" width="20" height="20" alt="an image"></amp-img></li>
+        <li> Nav item 5</li>
+        <li> Nav item 6</li>
+      </ul>
+    </amp-sidebar>
+
```
Comment (sebastianbenz) [inline src/20_Components/amp-sidebar.html]: Can you add a header: Sidebar Actions

**Comment by sebastianbenz** [comment]
Nice. Is it possible to change the closing behavior? e.g. clicking outside the menu doesn't close the sidebar?

**Comment by juliantoledo** [comment]
I don't think its possible to change the closing behavior.

**Comment by sebastianbenz** [comment]
LGTM

**Pull request merged**